node/src/test/scala/coop/rchain/node/configuration/hocon/HoconConfigurationSpec.scala

```diff
@@ -129,6 +129,7 @@ class HoconConfigurationSpec extends FunSuite with Matchers {
           numberOfActiveValidators = 100,
           genesisBlockNumber = 0,
           posMultiSigPublicKeys = GenesisBuilder.defaultPosMultiSigPublicKeys,
+          posVaultPubKey = GenesisBuilder.defaultPosVaultPubKey,
```

Comment:
Keep `posMultiSigPublicKeys` and `posMultiSigQuorum` parameters together. They are part of the same functionality.